src/styles/elements.less

```diff
@@ -26,3 +26,7 @@ h5 {
 h6 {
   font-size: @edu-heading-6-size;
 }
+
+textarea {
+  resize: none;
```

Comment:
This feels a bit misplaced to me here. I would move it into the component that reflects this property setup, in
```
.NeverScrollingTextArea {
   ...
  textarea {
      ...
  }
}
```

Reply:
We actually never want to see the ugly browser bars that appear for a fraction of a second when the server-rendered HTML is loaded. So I thought I'd put it into the global elements stylesheet like the `h1`-`h6` styles which fixed the problem. However, since this PR `textarea` is used in no other place than in `NeverScrollingTextArea`, so I moved it according to your suggestion